src/OpenLoco/src/Graphics/ImageId.h

```diff
@@ -57,112 +57,112 @@ namespace OpenLoco
 
         ImageId() = default;
 
-        explicit constexpr ImageId(ImageIndex index)
+        [[nodiscard]] explicit constexpr ImageId(ImageIndex index)
```

Comment:
So just reading up about this and turns out this does nothing. You need to move the [[nodiscard]] to the class/struct name L18

Reply:
Moving [[nodiscard]] to the class/struct name L18 does not work, ie generates an error in Visual Studio. 

Given my limited knowledge of C++ I don't think [[nodiscard]] is needed on the constructors because the ImageId struct does not have a return a value. You can assign the ImageId struct to an ImageId variable, ie 
```
ImageId imageId1{};
imageId1 = ImageId{ 0 };
```
ie you can assign a reference of ImageId struct, but you don't need to and should not cause a bug ie, you want a static variable ImageId unless you want to prevent that.

How would you like me to  proceed?

Reply:
Now removed  the [[nodiscard] from the constructors.